platform/qt/src/utils/geojson.cpp

```diff
@@ -1,69 +1,67 @@
-#include "qt_geojson.hpp"
+#include "geojson.hpp"
 
-#pragma clang diagnostic ignored "-Wenum-compare-switch"
```

Comment:
Interesting, was this not needed anymore?

Reply:
No, and it's good to rename ignores as soon as they are not needed anymore.